### DISCUSSION TITLE
Add search and filter to build new ride window

### DISCUSSION THREAD

**Discussion by chrismilne300** [OpenRCT2/OpenRCT2#15646] [Feature requests] [answered]
Not sure how useful it would be at the moment but further down the line as ride limits are removed, it would be handy to have a search/filter function in the new rides window. I can see this working like it does currently in the scenario editor.
![untitled-10](https://cloud.githubusercontent.com/assets/1381723/8266577/7e4413ae-172d-11e5-9042-64a88b09ec49.png)

**Comment by Broxzier** [comment]
This would also be helpful for the track selection, so you don't have to scroll through the entire list when there are many saved tracks.

**Comment by TheLarry98** [comment]
Yes please. This would be very useful.

**Comment by trzejos** [comment]
This would be very helpful especially when you have many ride objects selected

**Comment by trzejos** [comment]
I'm trying to implement this in #19067

**Comment by J-PIE-314** [comment, marked as answer]
The aforementioned PR #19067 was implemented, adding a search filter to the ride selection window.
